packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -699,12 +699,7 @@ export class Cluster extends Resource implements ICluster {
 /**
  * Options for adding worker nodes
  */
-export interface CapacityOptions extends autoscaling.CommonAutoScalingGroupProps {
-  /**
-   * Instance type of the instances to start
-   */
-  readonly instanceType: ec2.InstanceType;
-
+export interface CapacityOptions extends Omit<autoscaling.AutoScalingGroupProps, 'vpc' | 'machineImage' | 'updateType'> {
```

Comment:
Sadly jsii does not support `Omit`, so we will have to probably refactor the structs in `@aws-cdk/aws-autoscaling` to enable reuse.

Reply:
oh, TIL about [jsii](https://github.com/aws/jsii) :) so, what do you think is the best strategy? maybe just expand `CapacityOptions`?